### ISSUE TITLE
Project template `bin/setup` uses not-yet installed dependency

### ISSUE THREAD

**Issue by leonderijke** [kabisa/maji#223]
The default `bin/setup` script in the project template uses `cross-spawn` to kick off Yarn. However, this package is often not installed until _after_ running `bin/setup`, for example when cloning an existing project locally or in CI environments.

**Comment by pascalw** [comment]
Hm good point. We did this for Windows compatibility.

I think we can do one of the following:

1. Try to require `cross-spawn`, if that fails fall back to `spawn`.
2. Document that `cross-spawn` is a global dependency that should be installed first.
3. Get rid of `bin/setup`, since it's just an alias for `yarn install` now.

WDYT?

**Comment by leonderijke** [comment]
No. 3 sounds good to me. Alternatively, because `bin/setup` is solely there for backwards compatibility reasons we could also to revert `bin/setup` back to bash.
